gnomADc.pm

```diff
@@ -35,28 +35,58 @@ limitations under the License.
  or exome coverage files, available here:
 
  https://gnomad.broadinstitute.org/downloads
- To download the gnomad genomes coverage file in TSV format:
-  wget https://storage.googleapis.com/gcp-public-data--gnomad/release/2.1/coverage/genomes/gnomad.genomes.coverage.summary.tsv.bgz --no-check-certificate
-
- To download the gnomad exomes coverage file in TSV format: 
-  wget https://storage.googleapis.com/gcp-public-data--gnomad/release/2.1/coverage/exomes/gnomad.exomes.coverage.summary.tsv.bgz --no-check-certificate
+ To download the gnomad coverage file in TSV format:
+  for Assembly GRCh37: 
+   gnomad genomes:
+    wget https://storage.googleapis.com/gcp-public-data--gnomad/release/2.1/coverage/genomes/gnomad.genomes.coverage.summary.tsv.bgz --no-check-certificate
+   gnomad exomes:
+    wget https://storage.googleapis.com/gcp-public-data--gnomad/release/2.1/coverage/exomes/gnomad.exomes.coverage.summary.tsv.bgz --no-check-certificate
+  
+
+ To download the gnomad coverage file in TSV format: 
+  for Assembly GRCh38: 
+   gnomad genomes: 
+    wget https://storage.googleapis.com/gcp-public-data--gnomad/release/3.0.1/coverage/genomes/gnomad.genomes.r3.0.1.coverage.summary.tsv.bgz --no-check-certificate
+   gnomad exomes
+    wget https://storage.googleapis.com/gcp-public-data--gnomad/release/3.0.1/coverage/exomes/gnomad.exomes.r3.0.1.coverage.summary.tsv.bgz --no-check-certificate
+  
 
  The coverage summary files must be processed and Tabix indexed before
  use by this plugin. 
-
- The following steps are necessary to tabix the gnomad genomes coverage file :
- mv gnomad.genomes.coverage.summary.tsv.bgz gnomad.genomes.r2.1.gz
- gunzip gnomad.genomes.r2.1.gz
- sed '1s/.*/#&/'  gnomad.genomes.r2.1 > gnomad.genomes.tabbed.tsv
- bgzip gnomad.genomes.tabbed.tsv
- tabix -s 1 -b 2 -e 2 gnomad.genomes.tabbed.tsv.gz
-
- The following steps are neccessary to tabix the gnomad exomes coverage file :
- mv gnomad.exomes.coverage.summary.tsv.bgz gnomad.exomes.r2.1.gz
- gunzip gnomad.exomes.r2.1.gz
- sed '1s/.*/#&/'  gnomad.exomes.r2.1 > gnomad.exomes.tabbed.tsv
- bgzip gnomad.exomes.tabbed.tsv
- tabix -s 1 -b 2 -e 2 gnomad.exomes.tabbed.tsv.gz
+ 
+ for Assembly GRCh37:
+  The following steps are necessary to tabix the gnomad genomes coverage file :
+   mv gnomad.genomes.coverage.summary.tsv.bgz gnomad.genomes.r2.1.gz
+   gunzip gnomad.genomes.r2.1.gz
+   sed '1s/.*/#&/'  gnomad.genomes.r2.1 > gnomad.genomes.tabbed.tsv
+   bgzip gnomad.genomes.tabbed.tsv
+   tabix -s 1 -b 2 -e 2 gnomad.genomes.tabbed.tsv.gz
```

Comment:
We could make these steps simpler- the rename is not needed and we could avoid gunzip'ing to reduce space requirements, for example, in place of lines 59-61
gunzip -c gnomad.genomes.coverage.summary.tsv.bgz | sed '1s/.*/#&/'  > gnomad.genomes.tabbed.tsv